documentation.md

```diff
@@ -31,18 +31,27 @@ version: '1.16'
 ---------------------------
 
 <div class='row'>
-  <div class='col-xs-3 center'>
+  <div class='col-xs-4 center'>
```

Comment:
I think making part of the page 3 columns and part 4 doesn't look too good, I'd prefer having all icons on the same grid (but maybe that's just my OCD)

Reply:
The intention is to draw attention to (IMO) the three important icons on that page, given there are a lot now. Also 11 icons is a prime number, so there's no neat way to get them all on a single grid :)

Reply:
I'm going to merge as is, just to get this done, but I'll happily take a new PR for a better layout.